neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/embedded/impl/EmbeddedNeo4jGraphDatabaseFactory.java

```diff
@@ -64,32 +66,33 @@ public GraphDatabaseService create() {
 		AtomicBoolean isNew = new AtomicBoolean( false );
 		FactoryHolder factoryHolder = GRAPH_DATABASE_SERVICE_MAP.computeIfAbsent( dbLocationAbsolutePath, ( String dbPath ) -> {
 			LOG.infof( " Create new service instance for dbPath  %s", dbLocationAbsolutePath );
-			GraphDatabaseService service = buildGraphDatabaseService();
+			GraphDatabaseFacade service = buildGraphDatabaseService();
 			FactoryHolder holder = new FactoryHolder();
 			holder.setCounter( 1 );
-			holder.setGraphDatabaseService( service );
+			holder.setGraphDatabaseFacade( service );
 			isNew.set( true );
 			return holder;
 		} );
 		if ( !isNew.get() ) {
 			synchronized (GRAPH_DATABASE_SERVICE_MAP) {
-				boolean isAvailable = factoryHolder.getGraphDatabaseService().isAvailable( 100 );
+				factoryHolder.getGraphDatabaseFacade();
+				boolean isAvailable = factoryHolder.getGraphDatabaseFacade().isAvailable( 100 );
 				if ( isAvailable ) {
```

Comment:
What's 100?

Reply:
It is time of checking availability in milliseconds.

Reply:
It's better not to have Magic numbers like this in the codebase. This should be a constant called something like WAITING_TIME_MS

Reply:
fixed in next commit. A lot of thanks!